roles/cockpit/defaults/main.yml

```diff
@@ -0,0 +1,68 @@
+#########################################################################
+# Title:            Sandbox: cockpit | Default Variables                #
+# Author(s):        CHAIR/Raneydazed                                    #
+# URL:              https://github.com/saltyorg/Sandbox                 #
+# --                                                                    #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+################################
+# Basics
+################################
+
+cockpit_name: cockpit
+
+################################
+# Settings
+################################
+
+cockpit_vm_enabled: false
+cockpit_service_after: docker.service
+put_cockpit_dpkg_into_hold: true
+put_cockpit_machines_dpkg_into_hold: true
+
+################################
+# Paths
+################################
+
+cockpit_paths_socket_location: "/etc/systemd/system/cockpit.socket.d/listen.conf"
+cockpit_paths_socket_override_location: "/etc/systemd/system/cockpit.socket.d/override.conf"
+cockpit_paths_config_location: "/etc/cockpit/cockpit.conf"
+cockpit_paths_traefik_location: "/opt/traefik/cockpit.yml"
+cockpit_paths_service_location: "/lib/systemd/system/cockpit.service"
+cockpit_paths_override_location: "/etc/systemd/system/cockpit.service.d/override.conf"
+cockpit_paths_folders_list:
+  - "/etc/systemd/system/cockpit.socket.d"
+  - "/etc/systemd/system/cockpit.service.d"
+  - "/etc/cockpit"
+
+################################
+# Web
+################################
+
+cockpit_web_subdomain: "{{ cockpit_name }}"
+cockpit_web_domain: "{{ user.domain }}"
+cockpit_web_port: "1337"
+cockpit_web_url: "{{ 'https://' + (cockpit_web_subdomain + '.' + cockpit_web_domain
+                  if (cockpit_web_subdomain | length > 0)
+                  else cockpit_web_domain) }}"
+
+################################
+# DNS
+################################
+
+cockpit_dns_record: "{{ cockpit_web_subdomain }}"
+cockpit_dns_zone: "{{ cockpit_web_domain }}"
+cockpit_dns_proxy: "{{ dns.proxied }}"
+
+################################
+# Traefik
+################################
+
+cockpit_traefik_sso_middleware: "{{ traefik_default_sso_middleware }}"
+cockpit_traefik_middleware_default: "{{ traefik_default_middleware }}"
+cockpit_traefik_middleware_custom: ""
+cockpit_traefik_certresolver: "{{ traefik_default_certresolver }}"
+cockpit_traefik_enabled: true
```

Comment:
> Are these doing anything since the traffic handling is done via the file instead of labels? I don't see the middleware vars referenced in the traffic yml template. Should the `cockpit_traefik_enabled` toggle being set to `false` trigger removal of the cockpit.yml from the traefik folder?

hmm. I think so? I'll have to go over the role again to be sure, I used some vars in the yml template but you're prob right. I just stuck them in there without thinking about it. 

oo good. I like that.

but yeah I think that setting it to false should remove the yml template.

Reply:
@owine should be done here! hopefully :p